salt/standalone.sls

```diff
@@ -1,6 +1,6 @@
 {% from "salt/map.jinja" import salt_settings with context %}
 
-salt-minion:
+salt-minion-standalone:
```

Comment:
Just to understand:
* what is the problem with the previous ID? was it conflicting with another ID in the same formula? or in another formula?
* How is the new ID a better choice?

Reply:
Hi @daks, The error happens if salt combines `minion.sls` and `standalone.sls`, i.e. merged SLS files with identical ID is not allowed.  I was seeing problem on this branch https://github.com/saltstack-formulas/salt-formula/pull/421

Reply:
OK, fine. 

Aren't those two states designed to be incompatible? I mean `standalone.sls` is just `minion.sls` with a specific configuration, if I understand the README correctly.

If those are really incompatible and we don't want people to use both, maybe have an error is good, or maybe doc must be clear about that.

Reply:
Yes, they are definitely incompatible logic but so are conflicting IDs... both could be addressed but I'll keep this PR for the salt error.